### PR TITLE
[DSY-1000] Creates tag component

### PR DESCRIPTION
**Pull request by limamedeiros**
# Description

Creates tag component according to [documentation](https://xd.adobe.com/view/c44ffbe4-5883-4104-b959-4fc658a63dcb-911c/screen/0cafbe7c-474f-4135-9aff-05c5dc37f97e/). 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: SampleApp/Sources/Sample/Components/Tag/TagViewController.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import UIKit
+import NatDS
+
+class TagViewController: UIViewController, SampleItem {
+    static var name: String = "Tag"
+
+    private lazy var defaultAlert: NatTag = {
```
Comment (raonivaladares) [inline SampleApp/Sources/Sample/Components/Tag/TagViewController.swift:7]: Dúvida:

poderia ser um `let` aqui? pelo que entendi não uso do self.

Comment (limamedeiros) [reply SampleApp/Sources/Sample/Components/Tag/TagViewController.swift:7]: @raonivaladares pode sim, vou alterar.

**Review by gitrubs — APPROVED** [review]

**Review by raonivaladares — APPROVED** [review]
👍

**Comment by raonivaladares** [comment]
Thanks! o/

**Pull request merged**